src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -4,23 +4,89 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <ProjectGuid>{404DB891-B5AF-41E6-B89D-29E3F4573C4F}</ProjectGuid>
-    <!-- only supported by desktop CLR -->
-    <GeneratePlatformNotSupportedAssemblyMessage>SR.PlatformNotSupported_ReflectionContext</GeneratePlatformNotSupportedAssemblyMessage>
+    <!-- Only supported on .NET Framework and .NET Core -->
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetGroup)' == 'netstandard1.1' OR '$(TargetGroup)' == 'netstandard'">SR.PlatformNotSupported_ReflectionContext</GeneratePlatformNotSupportedAssemblyMessage>
     <!-- Xamarin frameworks claim to support PCL profiles which include this assembly,
          but do not include this assembly in their framework.
          This causes NuGet to give them the placeholder from the portable-* folder.
          Instead give them this assembly.  -->
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.1'">netstandard1.1;MonoAndroid10;MonoTouch10;xamarinios10;xamarintvos10;xamarinwatchos10</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.1'">netstandard1.1;$(AllXamarinFrameworks)</PackageTargetFramework>
```

Comment:
not sure this is necessary any longer. I would expect the xamarin platforms to automatically pick up the netstandard implementation now.